<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HumanReadableDataForm_template</class>
 <widget class="QDialog" name="HumanReadableDataForm_template">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string extracomment="Title of dialog to see the students statistics">Human-readable data</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTextEdit" name="textEdit"/>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Options</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="5" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QCheckBox" name="subjectsCheckBox">
           <property name="text">
            <string>Subjects</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="activityTagsCheckBox">
           <property name="text">
            <string>Activity tags</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QCheckBox" name="hoursCheckBox">
           <property name="text">
            <string>Hours</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="realHoursCheckBox">
           <property name="text">
            <string comment="MA mode = Mornings/Afternoons mode">Real hours (for MA mode)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="8" column="0">
        <widget class="QCheckBox" name="activitiesCheckBox">
         <property name="text">
          <string>Activities</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QCheckBox" name="timetableGenerationOptionsCheckBox">
         <property name="text">
          <string>Timetable generation options</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="fetVersionCheckBox">
           <property name="text">
            <string>FET version</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="modeCheckBox">
           <property name="text">
            <string>Mode</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="6" column="0">
        <widget class="QCheckBox" name="teachersCheckBox">
         <property name="text">
          <string>Teachers</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QCheckBox" name="daysCheckBox">
           <property name="text">
            <string>Days</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="realDaysCheckBox">
           <property name="text">
            <string comment="MA mode = Mornings/Afternoons mode">Real days (for MA mode)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="termsModeSettingsCheckBox">
         <property name="text">
          <string>Terms mode settings (for terms mode)</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QCheckBox" name="buildingsCheckBox">
           <property name="text">
            <string>Buildings</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="roomsCheckBox">
           <property name="text">
            <string>Rooms</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCheckBox" name="institutionNameCheckBox">
           <property name="text">
            <string>Institution name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="commentsCheckBox">
           <property name="text">
            <string>Comments</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="13" column="0">
        <widget class="QCheckBox" name="colorsCheckBox">
         <property name="text">
          <string>Colors</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QCheckBox" name="yearsCheckBox">
           <property name="text">
            <string>Years</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="groupsCheckBox">
           <property name="text">
            <string>Groups</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="subgroupsCheckBox">
           <property name="text">
            <string>Subgroups</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="10" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QCheckBox" name="timeConstraintsCheckBox">
           <property name="text">
            <string>Time constraints</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="spaceConstraintsCheckBox">
           <property name="text">
            <string>Space constraints</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="12" column="0">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin" stdset="0">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>604</width>
         <height>28</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveAsPushButton">
       <property name="text">
        <string>Save as</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
